.github/workflows/publish_to_maven_central.yml

```diff
@@ -15,9 +15,32 @@ on:
         type: string
 
 jobs:
-  publish_to_maven_central:
+  # This step should be removed when gradle plugin supports uploading to Central Portal
+  drop_all_open_ossrh_repositories:
     runs-on: ubuntu-latest
 
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
```

Comment:
I think it's better to create a `.python-version` file to store the version, to store the version in a dedicated file and not in the script. That way in the future we know where to look for the version, and it also unifies the python version we use throughout all our workflows. Also if we do that we don't have to specify this parameter, we can omit it. I think we can put it in the root directory, but not 100% sure, we can discuss further offline.

https://github.com/pyenv/pyenv?tab=readme-ov-file#understanding-python-version-selection